### DISCUSSION TITLE
Property Version of an NFT

### DISCUSSION THREAD

**Discussion by Pelumi527** [aptos-labs/aptos-developer-discussions#51] [Questions] [answered]
Hi, is there way to use the property version of an NFT to know if it is V1 or V2?

**Comment by gregnazario** [comment]
Property version is associated with V1, and doesn't apply to V2.

Additionally, if you're using the indexer, there should be a field for token standard which is "v1" or "v2".

cc @jillxuu

**Comment by jillxuu** [comment, marked as answer]
Yes, you could use indexer to figure out the token standard as we expose this field for tokens. An example query is [here](https://cloud.hasura.io/public/graphiql?endpoint=https%3A%2F%2Findexer.mainnet.aptoslabs.com%2Fv1%2Fgraphql&query=query+MyQuery+%7B%0A++token_activities_v2+%7B%0A++++token_standard%0A++++property_version_v1%0A++%7D%0A%7D%0A).

We also expose indexer queries through Aptos SDK, so that you could easily query it.

**Reply by Pelumi527** [reply]
Oh Ok, but I am using nft-scan api, and all the token has a property version
This a V2 token but it has a property_version
```
[
  {
    "collection": "GSU Mount",
    "creator": "0xa52530b10175571c5212cb8d2b1be8c125e4eec5a0768a3337ad4522476e1d80",
    "name": "Swift Black Horse",
    "asset_id": "0xa52530b10175571c5212cb8d2b1be8c125e4eec5a0768a3337ad4522476e1d80",
    "property_version": "1398",
    "tx_version": 208160475,
    "interact_function": "0xaaa6a5d82f58b3365c6c6ad59a336469695819fcee411dad02197115aeaba746::mount::mint_to",
    "minter": "0xe3f449ec7ca142fd9c851d9a00190cdf492defa312ca63eb9f372773b6951f9c",
    "owner": "0xc67545d6f3d36ed01efc9b28cbfd0c1ae326d5d262dd077a29539bcee0edce9e",
    "mint_timestamp": 1691082579000,
    "mint_transaction_hash": "0xa2265f3449cb54256e1b938dc664b8c25e6c794703f25ae89c1cb08115510e6f",
    "mint_price": 0,
    "token_uri": "https://live.asset.metapixel.io/nft/gsu_mount/black_horse.png",
    "metadata_json": null,
    "content_type": "image/png",
    "content_uri": "https://live.asset.metapixel.io/nft/gsu_mount/black_horse.png",
    "image_uri": "https://live.asset.metapixel.io/nft/gsu_mount/black_horse.png",
    "external_link": null,
    "latest_trade_price": null,
    "latest_trade_timestamp": null,
    "latest_trade_transaction_version": null,
    "latest_trade_transaction_hash": null,
    "attributes": null
  }
]
```

**Reply by gregnazario** [reply]
Not familiar with what NFT scan uses for that field, especially since 1398 is a really arbitrary number.

(Also good to see you're snooping on my NFTs :) )

**Reply by Pelumi527** [reply]
Oh alright thanks
😂😂 Oh didn't know it was your address, just picked a random address on Aptoscan
